### ISSUE TITLE
PowerVS VM provisioning request status error due to invalid "proc_type"

### ISSUE THREAD

**Issue by jaywcarman** [ManageIQ/manageiq-providers-ibm_cloud#148]
During VM provisioning status checks, if the Power Cloud API returns a VM with 'BUILD' status, then the included "proc_type" attribute is empty and the SDK raises an exception:

```
[EVM] VM [centosmiqtesttwo] Step [CheckProvisioned] Status [Error Creating VM] Message [[ArgumentError]: invalid value for "proc_type", must be one of ["dedicated", "shared", "capped"].]
```

The same exception would be raised if inventory refresh occurs if there is a VM with 'BUILD' status.

I believe this is ultimately a bug in the Power Cloud API backend and am following up with the API team internally within IBM. I'm not sure how long a fix would take, but there is also the possibility for an interim patch to https://github.com/IBM-Cloud/ibm-cloud-sdk-ruby (I'll open up an Issue/PR there as well).

**Comment by jaywcarman** [comment]
Fixed by https://github.com/IBM-Cloud/ibm-cloud-sdk-ruby/pull/49

I'll keep pushing for a back-end fix, though, so we can eliminate the change from the OpenAPI generated SDK.

**Comment by jaywcarman** [comment]
Verified issue in resolved with `ibm_cloud_power` v1.0.2
